lib/versioned/runner.js

```diff
@@ -97,7 +98,19 @@ function installPackages(cb) {
 function runTests(cb) {
   // TODO: Add tap arguments, such as color.
   process.send({ status: 'running' })
-  spawn('node', [testFile], function testHandler(err) {
+  let args = [testFile]
+  if (process.env.packageType && process.env.packageType === 'module') {
+    // The default loader is in the agent root, so this uses process.cwd() instead of __directory.
+    const cwd = process.cwd().slice(0, process.cwd().indexOf('/test/'))
+    let loaderPath = path.resolve(`${cwd}/`, 'esm-loader.mjs')
```

Comment:
I'm not sure there's a time we'd use the agent loader. At the very least it'd be a test loader.  In my [branch](https://github.com/bizob2828/node-newrelic/tree/esm-express) i called it `test/lib/test-loader.mjs`.  I was going to explore using testdouble to mock the agent API but regardless you'd need to call a loader  and if we used testdouble we wouldn't be able to call our loader with that so we'd have to chain them which is fine

Reply:
To avoid having to pass in the test loader I see we have a few options here:
 1. set the default to the agent's `test/lib/test-loader.mjs`
 2. move the test loader into the versioned repo and reference that as the default

Reply:
I verified passing in env var works:

```
loader=./test/lib/test-loader.mjs npm run versioned express-esm
```

Reply:
It makes sense to use the test loader. Default to `test/lib/test-loader.mjs` is fine, I'd think.